tests/framework/validators/DateValidatorTest.php

```diff
@@ -200,10 +195,6 @@ public function testValidateAttributeICUFormat($timezone)
             $model->attr_timestamp
         );
         $val = new DateValidator(['format' => 'yyyy-MM-dd']);
-        $model = FakedValidationModel::createWithAttributes(['attr_date' => []]);
-        $val->validateAttribute($model, 'attr_date');
-        $this->assertTrue($model->hasErrors('attr_date'));
-        $val = new DateValidator(['format' => 'yyyy-MM-dd']);
```

Comment:
I see what you mean now. If I recall correctly, these should cover robustness of the validator against array input.
The test should be kept for that reason and should be changed to test for non-empty array.